src/Mono.Android/Test/Mono.Android-Tests.csproj

```diff
@@ -125,4 +130,31 @@
   <ItemGroup>
     <Folder Include="Localization\" />
   </ItemGroup>
+  <ItemGroup>
+    <AndroidNativeLibrary Include="$(_ASANScript)" Condition=" '$(UseASAN)' != '' ">
```

Comment:
This is something users can use correct?
If so would it be possible for us to include a target which will auto add all this stuff based on a MSbuild boolean flag? like the `AndroidIncludeWrapSh` ? or `AndroidNativeCodeCheck`.

Just wondering if its worth the effort to make this easier for users to use. So they just enable one setting and we do the rest.

Reply:
The trouble is that `wrap.sh` can do anything - it can be used not only for sanitizers, but also for debugging, for setting Mono runtime environment variables, logging etc etc. We will not be shipping the checkers/sanitizers and so we won't be shipping the ASAN and UBSAN `wrap.sh` scripts - there's nothing to add automatically. 
The idea behind this mechanism here is that we want to be able to have the `AndroidNativeLibrary` items in the project but we don't want to package them into APK for normal builds. To include them users need to set `AndroidIncludeWrapSh`, probably from command line, for one-off builds when they need `wrap.sh` help for something.
Once I finish the gdb support we will include the gdb `wrap.sh` automatically the way you suggest, but until then there's nothing to include.